pkgs/development/python-modules/django-scheduler/default.nix

```
@@ -0,0 +1,46 @@
+{ lib
+, buildPythonPackage
+, django
+, fetchFromGitHub
+, pythonOlder
+, python-dateutil
+, pytz
+, icalendar
+}:
+
+buildPythonPackage rec {
+  pname = "django-scheduler";
+  version = "0.10.1";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "llazzaro";
+    repo = "django-scheduler";
+    rev = "refs/tags/${version}";
+    hash = "sha256-dY2TPo15RRWrv7LheUNJSQl4d/HeptSMM/wQirRSI5w=";
+  };
+
+  propagatedBuildInputs = [
+    django
+    python-dateutil
+    pytz
+    icalendar
+  ];
+
+  #Test is depricated
```

```suggestion
  # Test is deprecated
```
but you don't mean ``python setup.py test``?

---

Please try running https://github.com/llazzaro/django-scheduler/tree/develop/tests